connector/connect/client/jvm/src/test/scala/org/apache/spark/sql/PlanGenerationTestSuite.scala

```diff
@@ -1570,6 +1570,57 @@ class PlanGenerationTestSuite
     fn.hours(Column("a"))
   }
 
+  functionTest("make_dt_interval") {
+    fn.make_dt_interval(fn.col("a"), fn.col("a"), fn.col("a"), fn.col("b"))
+  }
```

Comment:
It seems we should add more test cases for `make_dt_interval`. But it will generate massive golden files.
I don't quite confirm it. cc @zhengruifeng

Reply:
it is fine to only add those files here. 
will not be too may I think

Reply:
The golden files are really many, haha